src/ansys/openapi/common/_oidc.py

```diff
@@ -181,61 +175,10 @@ def get_session_with_interactive_authorization(
             Number of seconds to wait for the user to authenticate. The default is ``60s``.
         """
 
-        async def await_callback() -> Any:
-            thread = threading.Thread(target=self._callback_server.serve_forever)
-            thread.daemon = True
-            thread.start()
-            return await self._callback_server.get_auth_code()
-
-        authorization_url, state = self._oauth_session.authorization_url(
-            self._well_known_parameters["authorization_endpoint"]
-        )
-        logger.info("Authenticating user...")
-        logger.debug(f"Opening web browser with URL {authorization_url}")
-        webbrowser.open(authorization_url)
-        loop = asyncio.get_event_loop()
-        auth_code_task = asyncio.wait_for(await_callback(), login_timeout)
-        auth_code = loop.run_until_complete(auth_code_task)
-        loop.close()
-        logger.info("Authentication complete, fetching token...")
-        if _log_tokens:
-            logger.debug(f"Received authorization code: {auth_code}")
-        self._callback_server.shutdown()
-
-        _ = self._oauth_session.fetch_token(
-            self._well_known_parameters["token_endpoint"],
-            authorization_response=auth_code,
-            include_client_id=True,
-            **self._idp_session_configuration,
-        )
-        if _log_tokens:
-            logger.debug(f"Access token: {self._oauth_session.token}")
-            if self._oauth_session.auto_refresh_url is not None:
-                # noinspection PyProtectedMember
-                logger.debug(
-                    f"Refresh token: {self._oauth_session._client.refresh_token}"
-                )
-        logger.info("Tokens retrieved successfully. Authentication complete.")
-        return self._oauth_session
-
-    def _configure_token_refresh(self) -> None:
-        """Configure automatic token refresh, if available.
-
-        This method only supports Authorization Code Flow currently.
-        """
-
-        def token_updater(token: Dict[str, str]) -> None:
-            self.token = token
-            self.access_token = token["access_token"]
-
-        logger.info("Refresh tokens supported, configuring...")
-        self._oauth_session.auto_refresh_url = self._well_known_parameters[
-            "token_endpoint"
-        ]
-        self._oauth_session.auto_refresh_kwargs = {
-            "client_id": self._authenticate_parameters["clientid"]
-        }
-        self._oauth_session.token_updater = token_updater
+        self._auth.timeout = login_timeout
+        self._authorized_session.auth = self._auth
+        self._authorized_session.get(self._api_url)
+        return self._authorized_session
```

Comment:
Glad to see all this code disappear! Since this is going, I'm guessing we can also delete OIDCCallbackHTTPServer, ResponseHandler, and all the tests for those classes?